app/main/views/auth.py

```diff
@@ -32,9 +33,14 @@ def render_login():
     next_url = request.args.get('next')
     if current_user.is_authenticated() and not get_flashed_messages():
         return redirect_logged_in_user(next_url)
+
+    form = LoginForm()
```

Comment:
just out of curiosity, what is the reason for first defining the form and only then passing it in?

Reply:
I need to reference the form in two places - on the line below to get errors from it, and then again to pass it into `render_template` :)

Reply:
Ahh, how did I not see that, cheers! :)